scripts/globals/conquest.lua

```diff
@@ -1069,9 +1069,11 @@ dsp.conquest.overseerOnEventFinish = function(player, csid, option, guardNation,
 
     -- SIGNET
     if option == 1 then
+        local ID = zones[player:getZoneID()]
```

Comment:
You don't this. mOffset (poor variable name) is already storing the text ID for CONQUEST

Reply:
I say mOffset is a poor variable name because technically it is storing the base ID for conquest messages. All of the `+ n` are the offsets. It's a little out of scope of this PR to go through and rename, but I just wanted to point that out.

Reply:
@isotor now that he pointed it out my OCD will make me change it later, so mOffset good'nuff